tests/PHPUnit/Framework/Fixture.php

```diff
@@ -388,6 +387,8 @@ public static function installAndActivatePlugins()
                 $pluginsManager->activatePlugin($name);
             }
         }
+
+        $pluginsManager->loadPluginTranslations();
```

Comment:
Previously it was calling `Piwik\Translate::loadAllTranslations()`, which was doing:

```php
Translate::loadCoreTranslation();
$pluginsManager->loadPluginTranslations();
```

You replaced it with `$pluginsManager->loadPluginTranslations()` only.

Could it be the source of the problem you had with translations? (i.e. core translations not loaded?)